Tweaked homepage layout and design

jidbujawe opened the pull request:
### This PR consist of the following changes:

- Changed the layout to include the images
- Tweaked the layout of the movies and celebs section
- Added a call-to-action section for user registration for guests

jidbujawe committed:
Tweaked homepage layout and design

jidbujawe committed:
Update home.blade.php

vrodriguero reviewed (commented):
The homepage looks awesome Jayd :D

Let me know when this is ready to merge

Great work

jidbujawe committed:
Update homepage for more styling

- Added a random movie so featured movie changes every refresh in homepage
- changed color of nav hover to blue
- changed footer bg to black

jidbujawe committed:
Fixed layout and spacing for all pages
